{"author":{"id":"ff90e8c1032712f76e3c119e253ae3a84614b47da224d1a3450b4f469b0523a9"},"ops":[{"type":3,"timestamp":1713563350,"nonce":"1BIpMne2C7XyfL47e90tTZWhaBU=","metadata":{"github-id":"IC_kwDOBLCY8M57ONh4","github-url":"https://github.com/svpcom/wfb-ng/issues/333#issuecomment-2067323000"},"message":"It seems that something broken in your python setup. Why it adds `local` to the path? Try to replace virtualenv to venv.\nI.e instead of `virtualenv --python=/usr/bin/python3 /home/daniel/wfb/wfb-ng/env` run `/usr/bin/python3 -m venv /home/daniel/wfb/wfb-ng/env`","files":null}]}